src/cargo/core/features.rs

```diff
@@ -935,6 +997,67 @@ impl GitoxideFeatures {
             internal_use_git2: false,
         }
     }
+
+    fn expecting() -> String {
+        let fields = vec!["'all'", "'fetch'", "'checkout'", "'internal-use-git2'"];
```

Comment:
ditto. And `internal-use-git2` shouldn't be suggested I feel like.

```suggestion
        let fields = ["'fetch'", "'checkout'"];
```